tests/foreman/ui_airgun/test_architecture.py

```diff
@@ -20,3 +20,10 @@ def test_positive_create_with_os(session):
             'operatingsystems.assigned': [os_name],
         })
         assert session.architecture.search(name) == name
+
+@parametrize('name', valid_data_list())
```

Comment:
Since tests are not members of the same class, there should be 2 empty lines between them, not 1.